### PR TITLE
Adding one cell on previous subsection only at right or left side

### PR DESCRIPTION
**Pull request by tadam50**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
When addCellForBusInfo parameter of BlockOrganizer is set to true, one cell is adding on previous subsection only at right and left side

**What is the new behavior (if this is a feature change)?**
Replace addCellForBusInfo parameter of BlockOrganizer by a map (bus id / side) 
in order to add cell on previous subsection only at right or left side

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DiagramLabelProvider.java [hunk 1]
```diff
@@ -90,4 +92,8 @@ public LabelPosition getPosition() {
     default Optional<BusInfo> getBusInfo(BusNode node) {
         return Optional.empty();
     }
+
+    default Map<String, Side> getBusInfoSides(VoltageLevelGraph graph) {
+        return Collections.emptyMap();
```
Comment (flo-dup) [inline single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DiagramLabelProvider.java:97]: In fact I thought of doing directly like:
`graph.getNodeBuses().forEach(busNode -> getBusInfo(busNode).ifPresent(busInfo -> result.put(busNode.getId(), busInfo.getAnchor())));`

**Review by flo-dup — CHANGES_REQUESTED** [review]

**Comment by flo-dup** [comment]
About the unit tests, could you test some corner-cases? like a subsection with no BusInfo at all. For instance remove the top and bottom in the middle (busInfo for bbs1 and bbs22) in one of the two unit tests.
